sdks/python/scripts/setup_nodejs.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+#
+# The script installs nodejs and needs to be executed before run_eslint and
+# run_jest.
+
+set -e
+
+if ! command -v node; then
+  if command -v apt-get; then
+    # From https://github.com/nodesource/distributions.
+    curl -sL https://deb.nodesource.com/setup_lts.x | sudo -E bash -
+    sudo apt-get install -y nodejs
```

Comment:
I am not ssure if we can run sudo on the jenkins workers? We can try and see what happens.

Reply:
Yes, we can run sudo on Jenkins workers.